h2o-py/h2o/h2o.py

```diff
@@ -23,6 +23,9 @@
 from .estimators.gbm import H2OGradientBoostingEstimator
 from .estimators.glm import H2OGeneralizedLinearEstimator
 from .estimators.xgboost import H2OXGBoostEstimator
+from .estimators.infogram import H2OInfogram
+from .estimators.deeplearning import H2OAutoEncoderEstimator, H2ODeepLearningEstimator
+from .estimators.extended_isolation_forest import H2OExtendedIsolationForestEstimator
```

Comment:
why so?
all those top level imports from `estimators.myalgo` already annoy me as they should not be here: those estimator classes are used only in examples and should therefore only be imported there, on demand.

Reply:
Done.